### ISSUE TITLE
Warning: Arguments to path.join must be strings Use --force to continue.

### ISSUE THREAD

**Issue by monolithed** [polyfillpolyfill/polyfill-service#340]
```
➜ git clone git@github.com:Financial-Times/polyfill-service.git
➜ cd polyfill-service 
➜ npm install

Running "installcollections" task
Cloning polyfill service source from git://github.com/financial-times/polyfill-service
> git clone git://github.com/financial-times/polyfill-service grunt-polyfill-service/node_modules/polyfill-service/polyfills/__repo
> git tag
1.0.0
1.0.1
v0.0.1
v1.0.0
v1.0.2
v1.1.0
v1.2.0
v1.3.0
v1.3.0-rc.1

> git checkout v0.0.1
> git checkout v1.0.0
> git checkout v1.0.2
> git checkout v1.1.0
> git checkout v1.2.0
> git checkout v1.3.0
Warning: Arguments to path.join must be strings Use --force to continue.

Aborted due to warnings.

npm ERR! Darwin 14.1.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE
npm ERR! polyfill-service@1.3.0-rc.1 prepublish: `grunt build`
npm ERR! Exit status 6
npm ERR! 
npm ERR! Failed at the polyfill-service@1.3.0-rc.1 prepublish script 'grunt build'.
npm ERR! This is most likely a problem with the polyfill-service package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     grunt build
npm ERR! You can get their info via:
npm ERR!     npm owner ls polyfill-service
npm ERR! There is likely additional logging output above.
```

**Comment by triblondon** [comment]
That was an issue in the rc, fixed in the 1.3.0 final I believe.

**Comment by triblondon** [comment]
Yup, was fixed in 846843588b2f9633d9ab2dbd46f5e6052e2afc32
